apps/funke/app/(app)/index.tsx

```diff
@@ -0,0 +1,70 @@
+import { useAppAgent } from '@/agent'
+import { addMessageListener } from '@animo-id/expo-ausweis-sdk'
+import { Button, Paragraph, XStack, YStack } from '@package/ui'
+import { Stack } from 'expo-router'
+import { useEffect, useState } from 'react'
+import { useSafeAreaInsets } from 'react-native-safe-area-context'
+import { ReceivePidUseCase, type ReceivePidUseCaseState } from '../../use-cases/ReceivePidUseCase'
+
+export default function Screen() {
+  const { top } = useSafeAreaInsets()
+  // FIXME: should be useReceivePidUseCase as the state is now not updated....
+  const [receivePidUseCase, setReceivePidUseCase] = useState<ReceivePidUseCase>()
+  const [state, setState] = useState<ReceivePidUseCaseState | 'not-initialized'>('not-initialized')
+  const [credential, setCredential] = useState<string>()
+  const { agent } = useAppAgent()
+
+  useEffect(() => {
+    const { remove } = addMessageListener((message) => {
```

Comment:
I think the ausweis sdk should have a more explicit name for this function or we should alias it here. Too vague imo.